primers_test.go

```diff
@@ -80,3 +81,50 @@ func TestMeltingTemp(t *testing.T) {
 		t.Errorf("MeltingTemp has changed on test. Got %f instead of %f", calcTM, expectedTM)
 	}
 }
+
+func ExampleDeBruijn() {
+	a := DeBruijn(4)
+
+	fmt.Println(a)
+	// Output: AAAATAAAGAAACAATTAATGAATCAAGTAAGGAAGCAACTAACGAACCATATAGATACATTTATTGATTCATGTATGGATGCATCTATCGATCCAGAGACAGTTAGTGAGTCAGGTAGGGAGGCAGCTAGCGAGCCACACTTACTGACTCACGTACGGACGCACCTACCGACCCTTTTGTTTCTTGGTTGCTTCGTTCCTGTGTCTGGGTGGCTGCGTGCCTCTCGGTCGCTCCGTCCCGGGGCGGCCGCGCCCCAAA
+}
+
+func ExampleUniqueSequence() {
+	c := make(chan string)
+	go UniqueSequence(c, 20, 4, []string{"CTCTCGGTCGCTCC"}, []func(string) bool{})
+	var output string
+	for a := range c {
+		output = a
+	}
+
+	fmt.Println(output)
+	// Output: TCTCGGTCGCTCCGTCCCGG
```

Comment:
Is there a reason why there is just one sequence outputted? What about "sorting for high/low GC, restriction enzyme sites" after using the channel to generate multiple sequences?

Reply:
@dexho can you respond to this and @Koeng101 can you maybe put a comment in this file about it?